src/server/db/migration/seed.ts

```diff
@@ -29,5 +30,10 @@ await db
   .values([{ id: 1, name: 'GPT-3.5 turbo', createdAt: new Date() }])
   .onConflictDoNothing()
 
+await db
```

Comment:
Because we added default value when adding column.
Don't think we need this.
And this is seed file, not migration. So, we shouldn't put migration code here as well.